modules/infra/gcp/bigquery.tf

```diff
@@ -0,0 +1,271 @@
+provider "google" {
+  project = var.gcp_project
+  region  = local.region
+}
+
+locals {
+  region = "northamerica-northeast2"
```

Comment:
It's often wise to pass things like region as arguments to the module, makes the module more reusable.